inc/saiqueue.h

```diff
@@ -80,6 +80,15 @@ typedef enum _sai_queue_attr_t
      */
     SAI_QUEUE_ATTR_INDEX = 0x00000001,
 
+    /**
+     * @brief Parent scheduler node
+     *
+     * @type sai_object_id_t
+     * @objects SAI_OBJECT_TYPE_SCHEDULER
+     * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
+     */
+    SAI_QUEUE_ATTR_PARENT_SCHEDULER_NODE = 0x00000002,
+
```

Comment:
Sorry, i did not realize we do not have port oid attribute in queue attribute list. 
SAI_QUEUE_ATTR_PARENT_SCHEDULER_NODE can take 2 objects, 
SAI_OBJECT_TYPE_SCHEDULER_GROUP  or SAI_OBJECT_TYPE_PORT. 
Please change comment.